gooddata-flight-server/gooddata_flight_server/cli.py

```diff
@@ -0,0 +1,132 @@
+#  (C) 2024 GoodData Corporation
+import argparse
+import sys
+import traceback
+from typing import Optional, Tuple, TypeVar
+
+from dynaconf import ValidationError
+
+from gooddata_flight_server.flexfun.flight_methods import (
+    create_flexfun_flight_methods,
+)
+from gooddata_flight_server.server.server_base import (
+    DEFAULT_LOGGING_INI,
+    ServerStartupInterrupted,
+)
+from gooddata_flight_server.server.server_main import (
+    GoodDataFlightServer,
+    create_server,
+)
+
+TConfig = TypeVar("TConfig")
+
+
+def _add_start_cmd(parser: argparse.ArgumentParser) -> None:
+    subcommands = parser.add_subparsers()
+    start_cmd = subcommands.add_parser("start")
+
+    start_cmd.add_argument(
+        "--config",
+        type=str,
+        nargs="*",
+        metavar="CONFIG_FILE",
+        help="Optionally specify one or more setting files to use. If not specified, all configuration has to be "
+        "provided using environment variables. Important: if you only specify this optional argument, "
+        "you must use the `--config-end` to explicitly terminate the list of files - otherwise the CLI cannot "
+        "correctly distinguish the end of list and will fail with usage error.",
+    )
+    start_cmd.add_argument(
```

Comment:
This is a pattern I haven seen before :-/ why not accept multiple --config values?

Reply:
```
    start_cmd.add_argument(
        "--config",
        type=str,
        nargs="*",
        metavar="CONFIG_FILE",
```

the --config is already setup with '*' number of arguments. the --config-end exists because otherwise the argparse may have problems going through something like this:

./cli --config a.toml b.toml <some other positional argument>

we do not use the positional arg now in this CLI, so this arg may be removed (this was from Quiver where we had say `./cli --config ... start --node something`)